client.go

```diff
@@ -151,5 +157,5 @@ func (c *Client) Call(ctx context.Context, method, path string, idempotencyKey s
 
 // Payment creates client for work with corresponding entity.
 func (c *Client) Payment() *PaymentClient {
```

Comment:
May be `NewPaymentClient`?

Reply:
It's like a REST approach. `client.Payment().Create()` looks good.

Reply:
It's the same as in our Zooz client, actually.

Reply:
Ok!